### PR TITLE
Add basic support for Github Actions

### PR DESCRIPTION
**Pull request by lexholden**

### CODE REVIEW

**Comment by lexholden** [comment]
@jjfreund @richardxia @lgarofalo this just needs the secret for the `TEXTELLENT_AUTH_CODE` to be added here https://github.com/ShelterTechSF/askdarcel-api/settings/secrets/actions and it the tests should pass, then we can merge this to replace Travis.

Still need a task to build/push the latest docker image on master, easy enough to add, will do separately.

**Review by lgarofalo — APPROVED** [review]

**Comment by lgarofalo** [comment]
I'll take care of `TEXTELLENT_AUTH_CODE` and update here when done.

**Comment by lgarofalo** [comment]
@lexholden Added PR to wire the secret env var from that link you sent into the build environment.

File: .github/workflows/images.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+      - uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - uses: docker/build-push-action@v2
```
Comment (lgarofalo) [inline .github/workflows/images.yml:23]: I think the step to tag the image is still missing, or is that done elsewhere?

Comment (lgarofalo) [reply .github/workflows/images.yml:23]: Ugh, I don't know why I didn't see, right below here it does this 🤦‍♂️

Comment (lexholden) [reply .github/workflows/images.yml:23]: Pretty sure this is all handled in this one action, you can see what this pushed to my personal docker hub here: https://hub.docker.com/repository/docker/lexholden/askdarcel-api

There is no version tag, it just pushes `latest`. You're right, we would probably need to support also running this on a release branch with the release name as a tag. Not sure how to do this, can find some time this weekend to figure it out if you think it's a blocker to merging this?

Comment (lgarofalo) [reply .github/workflows/images.yml:23]: https://github.com/docker/build-push-action/blob/master/docs/advanced/tags-labels.md

Looks like a good example for this

**Pull request merged**